reference/index.md

```diff
@@ -29,14 +29,5 @@ various APIs, CLIs, and file formats.
 | API                                                   | Description                                                                            |
 |:------------------------------------------------------|:---------------------------------------------------------------------------------------|
 | [Engine API](/engine/api/)                            | The main API for Docker, provides programmatic access to a daemon                      |
-| [Registry API](/registry/spec/api/)                   | Facilitates distribution of images to the engine                                       |
```

Comment:
Wondering if we need some page to describe that Docker Hub implements the OCI distribution spec (https://github.com/opencontainers/distribution-spec), otherwise we don't have documentation for the Registry API provided by Docker Hub.

Reply:
hey, I added a new page that briefly describes: 

a) what programmatic interfaces we have for Hub
b) the relation between Hub APIs and the protocol defined in distribution

wdyt? https://deploy-preview-15888--docsdocker.netlify.app/docker-hub/api/